nltk/test/unit/translate/test_bleu.py

```diff
@@ -22,7 +22,7 @@ def test__modified_precision(self):
         references = [ref1, ref2] 
         
         # Testing modified unigram precision.
-        hyp1_unigram_precision =  _modified_precision(references, hyp1, n=1)
+        hyp1_unigram_precision =  float(_modified_precision(references, hyp1, n=1))
         assert (round(hyp1_unigram_precision, 4) == 0.2857)
         # With assertAlmostEqual at 4 place precision.
         self.assertAlmostEqual(hyp1_unigram_precision, 0.28571428, places=4)
```

Comment:
I think it's okay to show that the `0.28571428` number doesn't necessarily need to be fit to the `places` parameter.